### PR TITLE
Find users in search list

### PR DESCRIPTION
**Pull request by MacMalainey**
Add functionality to be able to find users in search list

## Description of Changes

Add ability to find users by id
Add ability to find users by id in search list

## Checklist

- [ ] Automated tests
- [ ] Screenshots included (if necessary)
- [ ] Code is well commented

Closes #ISSUE_ID

### CODE REVIEW

File: src/app/src/main/java/com/cmput301f21t09/budgetprojectname/views/fragments/SearchFragment.java [hunk 1]
```diff
@@ -62,4 +78,15 @@ public void afterTextChanged(Editable s) {
         });
 
     }
+
```
Comment (j-vent) [inline src/app/src/main/java/com/cmput301f21t09/budgetprojectname/views/fragments/SearchFragment.java:81]: nit: add javadocs

**Review by j-vent — DISMISSED** [review]
LGTM!

**Review by Docker-J — APPROVED** [review]
LGTM!

**Review by cheetiantey — APPROVED** [review]
LGTM!

**Pull request merged**